### PR TITLE
Topic 02 exercise 03

### PR DESCRIPTION
**Pull request by fyunes**
Ex 03 Topic 03

### CODE REVIEW

File: Topic-02-Exercise-01/script.js [hunk 1]
```diff
@@ -41,10 +41,34 @@ class Movie extends EventEmitter {
         console.log(this.name, " Resume");
         this.emit('resume');
     }
+    addCast(cast) {
+        if (typeof cast === "object" && cast.length) {
```
Comment (DemianMoschin-GLB) [inline Topic-02-Exercise-01/script.js]: Why are you checking type here?

Comment (fyunes) [reply Topic-02-Exercise-01/script.js]: a security measure to check whats the input, it's not necessary in this case but thought it was useful to learn

Comment (DemianMoschin-GLB) [reply Topic-02-Exercise-01/script.js]: Please, remove

Comment (fyunes) [reply Topic-02-Exercise-01/script.js]: Done

**Comment by fyunes** [comment]
All done

File: Topic-02-Exercise-01/script.js [hunk 2]
```diff
@@ -42,11 +42,11 @@ class Movie extends EventEmitter {
         this.emit('resume');
     }
     addCast(cast) {
-        if (typeof cast === "object" && cast.length) {
+        if (cast.length) {
             cast.forEach(actor => {
                 this.castList.push(actor);
             });
-        } else if (typeof cast === "object" && cast.length === undefined) {
+        } else if (cast.length === undefined) {
```
Comment (DemianMoschin-GLB) [inline Topic-02-Exercise-01/script.js]: This can be refactored: why are you using else-if ?
You're asking a question twice

Comment (fyunes) [reply Topic-02-Exercise-01/script.js]: done

**Pull request merged**